x-pack/plugins/uptime/public/components/fleet_package/custom_fields.tsx

```diff
@@ -122,6 +124,34 @@ export const CustomFields = memo<Props>(
                   />
                 </EuiFormRow>
               )}
+              <EuiSpacer size="s" />
+              {isBrowser && (
```

Comment:
does it make sense to show this everytime user is adding a browser moniotor, is there a way to detect if they already have a working browser monitor, in that case i think user is probaly already educated and doesn;t need this additional context.

Reply:
That makes sense. There isn't a way to do this easily. We'd have to make a call to a fleet api to collect the users policies and check from there.  Alternatively, we could do a query against the datastream to see if it has any documents with `monitor.type` of browser.

It may be worth it to keep the call out though, since the user still need to make sure they are assigning the integration policy to the correct agent policy and they make have to change the selection at the bottom. Could be a good reminder.